### DISCUSSION TITLE
better user interface and lazy loading feature for better performance and fast loading .

### DISCUSSION THREAD

**Discussion by pankaj87072** [InternetHealthReport/gsoc#39] [General]
hey @m-appel @romain-fontugne I am analyzing the  loading speed factor of the website and the UI of web pages and i think one of the main factor of slow loading is the not optimized css and no use lazy loading which slow down the intial page loading.

**Comment by pankaj87072** [comment]
Before:-
![Screenshot from 2024-02-29 22-37-51](https://github.com/InternetHealthReport/gsoc/assets/112722707/0e521ab5-3b78-4fc7-b8d7-d76a3daa5095)
After:-
![Screenshot from 2024-02-29 22-40-48](https://github.com/InternetHealthReport/gsoc/assets/112722707/38cb32ec-2981-4ace-a020-f5927ac6ec83)
can we do something like this in all web pages for better look.

**Reply by pankaj87072** [reply]
should we focus on ui or not @m-appel @romain-fontugne

**Reply by dpgiakatos** [reply]
I don't think that UI is a priority in our GSoC topics.

**Reply by pankaj87072** [reply]
@dpgiakatos not only for gsoc I am into traceroute data visualization topic for gsoc

**Reply by romain-fontugne** [reply]
Indeed the website UI is not too important to us and our users. I'm closing this for now, please reopen if needed
